bufpipe.go

```diff
@@ -124,5 +124,6 @@ func (w *PipeWriter) CloseWithError(err error) error {
 		err = io.EOF
 	}
 	w.rerr = err
+	w.cond.Signal()
```

Comment:
I think this should be `Broadcast()`, don't you?

Reply:
Yes, updated!